### PR TITLE
{lib}[foss/2018b] LIBSVM v3.23

### PR DESCRIPTION
**Pull request by tstrempel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by tstrempel** [comment]
patch needed to create libsvm.so

File: easybuild/easyconfigs/l/LIBSVM/LIBSVM-3.23_shared_lib.patch [hunk 1]
```diff
@@ -0,0 +1,19 @@
+--- Makefile.orig	2019-11-04 10:10:12.475917486 +0100
```
Comment (smoors) [inline easybuild/easyconfigs/l/LIBSVM/LIBSVM-3.23_shared_lib.patch]: @tstrempel please add description and author to the top of the patch file?

**Comment by tstrempel** [comment]
> @tstrempel are these changes needed for the intel version of LIBSVM too? if so, please update that easyconfig as well.

I don't use the intel toolchain, so I can't say this for certain. 
I'll update the intel easyconfigs after this pull request is accepted.

**Comment by smoors** [comment]
I prefer to have them both in the same PR. If you add the updated intel version, I or someone else can test it for you.

File: easybuild/easyconfigs/l/LIBSVM/LIBSVM-3.23_shared_lib.patch (hunk 1 above)
Comment (tstrempel) [reply easybuild/easyconfigs/l/LIBSVM/LIBSVM-3.23_shared_lib.patch]: done

**Comment by tstrempel** [comment]
@smoors I updated the LIBSVM-3.23-intel-2018b.eb EasyConfig with the patch

**Comment by zao** [comment]
Test report by @zao
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
freja - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.17
See https://gist.github.com/3ab2c93a3b54faf1347832fcaa034601 for a full test report.

**Comment by zao** [comment]
Ignore the `intel/2018b` part of my test report, the machine cannot run such an old Intel compiler 🤡

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node377.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/614953faf852e1916d1276dbbc0911de for a full test report.

**Comment by smoors** [comment]
Going in, thanks @tstrempel!

**Pull request merged**